torchgeo/datasets/globbiomass.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         self,
         paths: Path | Iterable[Path] = 'data',
         crs: CRS | None = None,
-        res: float | None = None,
+        res: tuple[float, float] | None = None,
```

Comment:
All of these other datasets also need to be `float | tuple[float, float]` and need updated docstrings too.